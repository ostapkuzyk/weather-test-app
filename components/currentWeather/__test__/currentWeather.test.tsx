import CurrentWeather from '../index';
import { render } from '@testing-library/react-native';

const mockProps = {
  temp: 0,
  feelTemp: 0,
  dewPoint: 0,
  humidity: 0, 
  windSpeed: 0
}

describe('currentWeather', () => {
  it('renders without crashing', () => {
    const comp = render(<CurrentWeather {...mockProps} />);
    expect(comp.toJSON()).toMatchSnapshot();
  });
});