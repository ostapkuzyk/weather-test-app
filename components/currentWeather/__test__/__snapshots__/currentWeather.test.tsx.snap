// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currentWeather renders without crashing 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "justifyContent": "center",
      "marginTop": -24,
      "width": "100%",
    }
  }
  testID="currentWeather"
>
  <Text
    style={
      {
        "color": "white",
        "fontSize": 64,
      }
    }
  >
    0
    °
  </Text>
  <Text
    style={
      {
        "color": "white",
        "fontSize": 20,
        "marginLeft": 6,
      }
    }
  >
    Feels like: 
    0
    °
  </Text>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "rgba(16,35,70,0.8)",
        "borderRadius": 20,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginTop": 12,
        "paddingHorizontal": 10,
        "paddingVertical": 7,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="28"
        bbWidth="28"
        fill="white"
        focusable={false}
        height="28"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 28,
              "width": 28,
            },
          ]
        }
        vbHeight={1024}
        vbWidth={1024}
        width="28"
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4294967295,
              "type": 0,
            }
          }
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M318.593 376.702c43.367-17.548 66.566-65.743 48.949-109.28-17.566-43.366-135.491-104.486-135.491-104.486s-42.327 126-24.772 169.355c17.623 43.543 67.823 62.018 111.315 44.405v0.006z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M500.221 633.506c0 0-43.533 122.654-25.984 166.003 17.613 43.494 67.691 62.067 111.189 44.454 43.49-17.604 66.543-65.783 48.939-109.274-17.557-43.366-134.144-101.188-134.144-101.188v0.004z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M227.807 584.469c0 0-47.759 132.113-28.852 178.82 18.954 46.822 73.97 67.273 120.799 48.307 46.703-18.906 68.595-70.473 49.635-117.308-18.921-46.694-141.582-109.815-141.582-109.815v-0.004z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M499.197 544.448c26.662-10.795 37.257-42.227 27.793-65.6-10.846-26.795-79.164-61.287-79.164-61.287s-25.114 72.32-14.272 99.128c9.463 23.36 38.899 38.588 65.643 27.759v0z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M760.915 533.342c-10.846-26.79-79.164-61.286-79.164-61.286s-25.114 72.324-14.268 99.127c9.463 23.369 38.903 38.596 65.638 27.772 26.667-10.807 37.269-42.24 27.793-65.613v0z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M708.767 275.428c-17.557-43.36-135.488-104.48-135.488-104.48s-42.325 126-24.772 169.355c17.63 43.543 67.827 62.006 111.317 44.4 43.354-17.542 66.564-65.726 48.943-109.279v0.004z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M826.071 791.906c-10.846-26.795-79.164-61.286-79.164-61.286s-25.114 72.32-14.268 99.127c9.463 23.364 38.895 38.592 65.638 27.759 26.658-10.795 37.257-42.227 27.793-65.6v0z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          {
            "color": "white",
            "fontSize": 20,
            "marginLeft": 6,
          }
        }
      >
        0
      </Text>
    </View>
    <View
      style={
        {
          "width": 26,
        }
      }
    />
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="28"
        bbWidth="28"
        fill="white"
        focusable={false}
        height="28"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 28,
              "width": 28,
            },
          ]
        }
        vbHeight={1024}
        vbWidth={1024}
        width="28"
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4294967295,
              "type": 0,
            }
          }
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M519.519 706.955c-10.697-9.323-19.452-28.561-19.452-42.756v-493.129c0-14.188-11.605-25.795-25.792-25.795h-33.088c-14.187 0-25.792 11.606-25.792 25.795l-0.009 493.117c0 14.199-9.102 33.011-20.222 41.822 0 0-47.531 37.641-47.531 73.229 0 54.942 48.349 99.486 107.988 99.486 59.631 0 107.972-44.548 107.972-99.486 0-33.882-44.075-72.29-44.075-72.29v0.009z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M489.358 288.561c0 14.189-11.605 25.795-25.796 25.795h-11.298c-14.187 0-25.796-11.606-25.796-25.795l-0.004-106.291c0-14.188 11.605-25.795 25.796-25.795h11.298c14.187 0 25.792 11.606 25.792 25.795l0.009 106.291z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M628.9 597.592c0 24.363-19.733 44.1-44.109 44.1-24.38 0-44.105-19.729-44.105-44.1l43.511-79.39 44.702 79.39z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M676.366 669.754c-0.141 14.14-11.695 25.485-25.852 25.348-14.118-0.128-25.472-11.686-25.331-25.835l25.69-45.828 25.493 46.315z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M667.078 556.615c-0.073 8.431-6.972 15.198-15.394 15.108-8.418-0.077-15.202-6.972-15.108-15.394l15.326-27.324 15.177 27.61z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M518.598 799.67c-0.841-0.013-0.107-0.017 1.626-0.017 1.707 0-2.014 10.423-8.286 23.147l-2.048 4.143c-6.263 12.727-20.715 23.151-32.102 23.151h-41.434c-11.388 0-25.84-10.423-32.114-23.151l-2.046-4.143c-6.275-12.723-9.977-23.147-8.24-23.147 1.749 0 14.782 0 28.973 0.013l71.415 0.009c14.178 0 25.105-0.004 24.256-0.004v0z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M447.677 788.491c-16.439-0.947-25.839-6.251-34.249-22.199-8.413-15.966-7.401-22.101 1.084-40.768l0.521 0.060c19.806 3.546 25.387 6.281 33.702 22.127 8.439 15.936 7.475 26.709-1.062 40.772z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M466.945 743.026c-7.326-0.414-11.516-2.782-15.258-9.882s-3.298-9.843 0.491-18.15l0.218 0.017c8.823 1.583 11.315 2.812 15.002 9.856 3.746 7.091 3.319 11.9-0.482 18.163z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M476.396 788.491c-7.479-9.182-9.229-16.764-4.834-28.988 4.416-12.215 8.162-14.643 22.199-19.204l0.201 0.32c7.748 12.233 8.977 16.533 4.634 28.634-4.39 12.215-10.594 16.913-22.199 19.204z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          {
            "color": "white",
            "fontSize": 20,
            "marginLeft": 6,
          }
        }
      >
        0
      </Text>
    </View>
    <View
      style={
        {
          "width": 26,
        }
      }
    />
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="28"
        bbWidth="28"
        fill="white"
        focusable={false}
        height="28"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "flex": 0,
              "height": 28,
              "width": 28,
            },
          ]
        }
        vbHeight={1024}
        vbWidth={1024}
        width="28"
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4294967295,
              "type": 0,
            }
          }
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M167.772 451.857h327.564c39.629 0.286 77.969-14.071 107.661-40.324 29.692-26.251 48.649-62.543 53.222-101.908 4.412-44.212-10.24-88.212-40.286-120.948-29.828-32.749-72.051-51.446-116.348-51.52h-88.388c-14.217 0-27.353 7.588-34.463 19.898-7.109 12.314-7.109 27.486 0 39.8 7.109 12.309 20.246 19.898 34.463 19.898h88.388c22.4 0.029 43.703 9.714 58.466 26.565 14.754 16.852 21.542 39.246 18.615 61.457-2.931 19.057-12.681 36.4-27.439 48.806-14.754 12.411-33.515 19.034-52.791 18.651h-328.672c-14.217 0-27.354 7.583-34.464 19.898s-7.109 27.48 0 39.794c7.109 12.314 20.246 19.896 34.464 19.896l0.007 0.038z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M895.264 714.41c-4.574-39.364-23.522-75.652-53.21-101.897-29.692-26.253-68.023-40.619-107.652-40.337h-273.626c-14.221 0-27.354 7.582-34.466 19.9-7.108 12.314-7.108 27.486 0 39.791 7.108 12.318 20.245 19.9 34.466 19.9h274.709c19.273-0.384 38.033 6.246 52.787 18.65 14.763 12.412 24.508 29.756 27.439 48.806 2.927 22.212-3.857 44.604-18.615 61.457-14.763 16.858-36.062 26.539-58.462 26.573h-88.422c-14.221 0-27.354 7.582-34.466 19.895-7.108 12.309-7.108 27.482 0 39.795s20.245 19.9 34.466 19.9h88.38c44.318-0.043 86.575-18.726 116.42-51.482 30.033-32.742 44.672-76.74 40.243-120.947l0.009-0.004z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M373.083 612.010c-0.006-10.56-4.2-20.681-11.657-28.147-7.463-7.471-17.583-11.674-28.137-11.686h-165.522c-14.217 0-27.354 7.582-34.463 19.9-7.109 12.314-7.109 27.486 0 39.791 7.108 12.318 20.246 19.9 34.463 19.9h165.522c10.543-0.013 20.651-4.207 28.109-11.657 7.458-7.454 11.663-17.557 11.686-28.096l-0.001-0.004z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M670.061 477.559c3.396 0 6.652-1.348 9.050-3.746l26.97-26.974c37.069 7.351 75.302-4.983 101.094-32.599 25.796-27.617 35.486-66.6 25.626-103.081-1.178-4.434-4.642-7.897-9.075-9.074-36.48-9.857-75.465-0.166-103.083 25.623-27.614 25.795-39.949 64.029-32.597 101.097l-27.034 26.891c-3.657 3.665-4.753 9.165-2.773 13.948 1.984 4.783 6.647 7.898 11.823 7.906v0.009z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
          <RNSVGPath
            d="M393.842 705.663l-26.972-26.974c-5.023-4.851-13.006-4.774-17.943 0.162s-5.006 12.919-0.154 17.941l26.955 26.97c-6.943 35.695 4.223 72.508 29.821 98.33 26.624 26.347 86.135 48.717 114.944 19.2h0.009c9.843-36.467 0.158-75.43-25.613-103.044-25.779-27.618-63.991-39.953-101.047-32.636v0.051z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
            propList={
              [
                "stroke",
                "strokeWidth",
              ]
            }
            stroke={
              {
                "payload": 4294967295,
                "type": 0,
              }
            }
            strokeWidth={5}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          {
            "color": "white",
            "fontSize": 20,
            "marginLeft": 6,
          }
        }
      >
        0.0
      </Text>
    </View>
  </View>
</View>
`;
